app/src/test/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegrationTest.kt

```diff
@@ -107,6 +108,24 @@ class BrowserToolbarIntegrationTest {
         verify(browserToolbarIntegration).observerSecurityIndicatorChanges()
     }
 
+    @Test
+    fun `WHEN start method is called THEN observe erase tabs CFR changes`() {
+        doNothing().`when`(browserToolbarIntegration).observeEraseCfr()
+
+        browserToolbarIntegration.observeEraseCfr()
+
+        verify(browserToolbarIntegration).observeEraseCfr()
```

Comment:
The `verify` here seems redundant.

To actually test that when  `state.showEraseTabsCfr` changes we'll try showing the intended CFR (can't test that we actually show one because the CFR instance is internally constructed) the code might be refactored such that:
- when the state changes we call a method to build the CFR  - this can then be tested
- the method to build the CFR can be tested to ensure all properties are the intended ones - can test container/text/anchor/dismiss callback.
- there is another method which receives a CFR instance and just calls `show()` on it. This can also be tested.